crates/bevy_tasks/src/lib.rs

```diff
@@ -34,10 +36,5 @@ pub mod prelude {
     };
 }
 
-pub fn logical_core_count() -> usize {
-    num_cpus::get()
-}
-
-pub fn physical_core_count() -> usize {
-    num_cpus::get_physical()
-}
+pub use num_cpus::get as logical_core_count;
+pub use num_cpus::get_physical as physical_core_count;
```

Comment:
FYI the docs of this mentions that the fallback is `get()` but bevy renames the import. Not a biggie, as the link should still work and point to the function in the num_cpus crate.

Reply:
I think this is fine since the alias is made evident in the docs.